styles/colors.js

```diff
@@ -0,0 +1,8 @@
+const colors = {
+  dark: '#151616',
```

Comment:
Same here be more specific what does this colour actually represent. Background, text colour?